### DISCUSSION TITLE
Limitations to mapping large geomaps ?

### DISCUSSION THREAD

**Discussion by dosfslib** [grafana/grafana#66537] [Dashboards, Panels & Visualizations]
Hello, 

I was wondering if there limitations to mapping large data set of location in the geomap plugin. 
Tried to map more than 6-8 k location points and the memory usage ends up being quite large in memory i.e. it's grabbing at least 6 Gigs of my gpu memory.... 

Is there a way to optimize it otherwise ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
